proto/process/agent.proto

```diff
@@ -117,11 +117,12 @@ message CollectorReqStatus {
 
 message CollectorPod {
 	string hostName = 1;
-	repeated Pod pods = 2;
-	int32 groupId = 3;
-	int32 groupSize = 4;
+	string clusterName = 2;
```

Comment:
Nothing on the backend is processing these protobufs are they? Otherwise you'll want this to be `7` instead of `2`

Reply:
not yet 👍